include/session.php

```diff
@@ -40,12 +40,12 @@ function ref_session_write ($id,$data) {
 	if($session_exists) {
 		$r = q("UPDATE `session`
 				SET `data` = '%s'
-				WHERE `data` != '%s' AND `sid` = '%s'",
+				WHERE `sid` = '%s' AND `data` != '%s'",
 				dbesc($data), dbesc($data), dbesc($id));
```

Comment:
Shouldn't it rather be `dbesc($data), dbesc($id), dbesc($data));` because you have changed the ordering of `sid` and `data` in the line above?